pkg/kubelet/kubelet.go

```diff
@@ -339,6 +399,32 @@ func (kl *Kubelet) setupDataDirs() error {
 	return nil
 }
 
+func (kl *Kubelet) setupTmpfsDataDirs() error {
+	kl.tmpfsRootDirectory = path.Clean(kl.tmpfsRootDirectory)
+	// TODO: handle edge cases like mount already exists
```

Comment:
The reason we changed to !windows was OSX and unit tests, if I recall.

If the tmpfs is a function of the root directory we can't blindly unmount/remount it - it will have been bind-mounted into pods, so either we won't be allowed to unmount it or we will totally screw over any running pods.  This is important - kubelet MUST be able to crash and restart without killing pods.